flocker/control/schema/types.yml

```diff
@@ -217,8 +217,10 @@ definitions:
     # on what this can be set to for the ZFS backend.  Could probably be
     # considered more carefully.
     minimum: 67108864
-    # This is how you require integers, of course.
-    divisibleBy: 1
+    # Filesystem initialization in the block device backend requires sizes to
+    # be a multiple of 512.
+    multipleOf: 512
```

Comment:
I suggest making this 1024, insofar as you've just added code that presumes KiB.